.gitignore

```diff
@@ -1,2 +1,3 @@
 **/node_modules
-config/config.js
\ No newline at end of file
+config/config.js
+.env
```

Comment:
Do we have a .env file? Saw that we have a database password in the docker compose file, so I was wondering if we have a .env file with that password we should share (not here of course).

Reply:
The .env file is a file we'd each just create manually ourselves, since the file is literally just one line of text that says "DB_PASSWORD=<password>". The passwords don't have to be the same either - at least for testing - it's just what we'd use to log in when running the system.